roles/load_aware_deploy_trimaran/tasks/main.yml

```diff
@@ -47,27 +47,33 @@
   retries: 30
   until: token_size.stdout | int > 1000 
 
+- name: Configure Trimaran options
+  template:
+    src: "{{ trimaran_setup_template }}"
+    dest: "{{ trimaran_setup_config }}"
+
```

Comment:
please add here something like this:
```
- name: Create the src artifacts directory
  file:
    path: "{{ artifact_extra_logs_dir }}/src/"
    state: directory
    mode: '0755'

- name: define the variables
  set_fact:
    trimaran_setup_config: "{{ artifact_extra_logs_dir }}/src/trimaran-templated-config.yaml"

- name: Configure Trimaran options
  template:
    src: "{{ trimaran_setup_template }}"
    dest: "{{ trimaran_setup_config }}"
```
(the `set_fact` task isn't mandatory, it's just to define a variable/constant instead of having a harcoded path duplicated here and there. In theory that's preferable, but in practice I often omit it :#)